clang-tools-extra/clang-doc/Representation.h

```diff
@@ -535,6 +535,8 @@ struct ClangDocContext {
   // JavaScript files that will be imported in all HTML files.
   std::vector<std::string> JsScripts;
   StringRef Base;
+  // Mustache Template files
```

Comment:
This is a map from strings to strings. What is the semantics of that map? It's not obvious to me from this change, and it should probably be part of the comment.

Reply:
My understanding is that it provides a mapping to of a template "type" (e.g., a comment-template) to a real mustache file. I'll update the comment. I'll need to look further up the stack to confirm.

Reply:
done. LMK if something is unclear.